endpoints/bookstore-grpc/README.md

```diff
@@ -0,0 +1,42 @@
+# Google Cloud Endpoints Bookstore App in Python
+
+## Installing the dependencies using virtualenv:
+
+    virtualenv bookstore-env
+    source bookstore-env/bin/activate
+
+Install all the Python dependencies:
+
+    pip install -r requirements.txt
+
+Install the [GRPC libraries and tools](https://github.com/grpc/grpc/blob/v1.0.x/INSTALL.md)
```

Comment:
Follow prequisites in [GRPC Quickstart](http://www.grpc.io/docs/quickstart/python.html#prerequisites) is better than the INSTALL.md